docs/output.md

```diff
@@ -455,7 +455,9 @@ Besides the reference files or output files created by BUSCO, the following summ
 
 </details>
 
-### QC for metagenome assembled genomes with CheckM
+### QC for metagenome assembled genomes
```

Comment:
Wouldnt that headline be better at line 419 (`### QC for metagenome assembled genomes with BUSCO`) followed by `#### BUSCO` ?